daal4py/sklearn/_utils.py

```diff
@@ -362,7 +318,9 @@ def get_suggested_n_threads(n_cpus):
 
 
 def run_with_n_jobs(method):
```

Comment:
And what about tests for this method? We should add least add the most basic closures, like checking if attributes that should be added are there, functions are decorated, etc.

Reply:
Basic test to check sklearnex logs was added.